mava/utils/environments/pettingzoo_utils.py

```diff
@@ -130,7 +138,12 @@ def make_environment(
     else:
         raise Exception("Pettingzoo is not installed.")
 
+    # This should not be used with petting zoo environments since
+    # stacking is already done using supersuit.
+    if stack_frames > 1:
```

Comment:
Are we going to implement/import observation stacking from Supersuit in another PR? If so, maybe we should just throw a `NotImplementedException` here for now.

Reply:
I added this comment because we have [[this](https://github.com/instadeepai/Mava/blob/feature/frame_stacking/mava/utils/environments/pettingzoo_utils.py) in the atari pre-processing that already does the stacking. But I will prefer we have single approach to implement these wrappers as you suggested similar to supersuit